packages/sanity/src/desk/structureBuilder/MenuItem.ts

```diff
@@ -285,7 +285,14 @@ export function getOrderingMenuItem(
 ): MenuItemBuilder {
   let builder = new MenuItemBuilder(context)
     .group('sorting')
-    .title(`Sort by ${title}`) // fallback title
+    .title(
+      context.i18n.t('default-menu-item.fallback-title', {
+        // note this lives in the `studio` bundle because that one is loaded by default
+        ns: 'studio',
+        defaultValue: `Sort by ${title}`, // fallback value
```

Comment:
Do we still need this defaultValue if we have `default-menu-item.fallback-title` ?